src/test/java/org/apache/commons/text/ExtendedMessageFormatTest.java

```diff
@@ -416,6 +416,81 @@ public void testSetFormatsByArgumentIndex() {
         emf.setFormatsByArgumentIndex(new Format[]{new LowerCaseFormat(), new UpperCaseFormat()});
     }
 
+    @Test
+    public void testFailsToCreateExtendedMessageFormatTakingThreeArgumentsThrowsIllegalArgumentExceptionOne() {
+        Map<String, FormatFactory> map = new HashMap<String, FormatFactory>();
```

Comment:
As commons-text requires Java 7 this can be reduced to `Map<String, FormatFactory> map = new HashMap<>();`.